app/services/send_applications_to_provider.rb

```diff
@@ -4,6 +4,7 @@ def call
     GetApplicationFormsReadyToSendToProviders.call.each do |application_form|
       application_form.application_choices.each do |choice|
         SendApplicationToProvider.new(application_choice: choice).call
+        SendNewApplicationEmailToProvider.new(application_choice: choice).call
```

Comment:
Sorry to be late to the party here, but this probably belongs inside the `SendApplicationToProvider` class too!